libstuff/libstuff.h

```diff
@@ -242,28 +239,23 @@ inline void SLogLevel(int level) {
     _g_SLogMask = LOG_UPTO(level);
     setlogmask(_g_SLogMask);
 }
-inline bool SLogLevelIsSet(int level) { return (_g_SLogMask & (1 << level)); }
 
 // Stack trace logging
-extern void SLogStackTrace();
+void SLogStackTrace();
 
 // Simply logs a stream to the debugger
 // **NOTE: rsyslog max line size is 2048 bytes.  We split on 1500 byte bounderies in order to fit the
 //         syslog line prefix and the expanded \r\n to #015#012
 // **FIXME: Everything submitted to syslog as WARN; doesn't show otherwise
-extern bool _g_SLogToSTDOUT;
 #define SSYSLOG(_PRI_, _MSG_)                                                                                          \
     do {                                                                                                               \
         SThreadLocalStorage* tls = SThreadGetLocalStorage();                                                           \
         if (_g_SLogMask & (1 << (_PRI_))) {                                                                            \
             ostringstream __out;                                                                                       \
             __out << _MSG_ << endl;                                                                                    \
             const string& __s = __out.str();                                                                           \
-            if (_g_SLogToSTDOUT)                                                                                       \
```

Comment:
Similarly, the ability to log to STDOUT has also been pretty helpful in the past, generally when using `libstuff` to develop some command-line application.  I think we've removed most of those over the years, but I know we had several back in the day. Anyway, I'm fine removing it, it's easy to bring back if we care to.